ParisBNF/abb/BNFabb92.xml

```diff
@@ -0,0 +1,682 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb92" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Acts of saints and martyrs, and homilies for
+                    various festivals assigned for the month <foreign xml:lang="gez"
+                    >Ḫǝdār</foreign></title>
+                <title xml:lang="gez" xml:id="t2">መጽሐፈ፡ ኅዳር፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Maṣḥafa Ḫǝdār</title>
+                <title xml:lang="fr" xml:id="t3">Melanges théologiques </title>
+                <title xml:lang="fr" xml:id="t4" resp="PRS3082Chaine">Récits
+                    hagiographiques.</title>
+                <editor key="NV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b531387571">BnF Éthiopien
+                            d'Abbadie 92</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
```

Comment:
`   <textLang mainLang="gez"/>` is not present in all main msItems (it is not needed in the nested ones if present in a parent)

Reply:
> <textLang mainLang="gez"/>

this I do not understand! why do we need this empty element? I once asked and no one explained me, so I stopped using it.

Reply:
It indicates that the work is in Geez